Haneke/UIButton+Haneke.swift

```diff
@@ -37,16 +37,13 @@ public extension UIButton {
     }
     
     public func hnk_setImageFromFetcher(fetcher : Fetcher<UIImage>, state : UIControlState = .Normal, placeholder : UIImage? = nil, format : Format<UIImage>? = nil, failure fail : ((NSError?) -> ())? = nil, success succeed : ((UIImage) -> ())? = nil){
+        self.setImage(placeholder, forState: state)
```

Comment:
Hi sorry I just saw your comment. Would you mind expanding a little bit? How can I set the placeholder or nil out the image?
I'm using hnk_setImageFromURL(profilePicURL) so I can't really deal with the placeholder